bcda/cli.go

```diff
@@ -309,7 +345,9 @@ func importCCLFDirectory(filePath string) (success, failure, skipped int, err er
 	}
 
 	if len(cclfmap) == 0 {
-		return 0, 0, 0, errors.New("failed to find any CCLF files in directory")
+		err := errors.New("failed to find any CCLF files in directory")
+		log.Error(err)
```

Comment:
Would it be helpful to print this in addition to logging it?

Reply:
yes and it does print because of the line afterwards. The return there gets us back to the invoking main.go where it bubbles up to stdout.